ansible_roles/drools/files/drools

```diff
@@ -31,6 +32,10 @@ case "$1" in
     if is_running; then
         echo "Already started"
     else
+        rm -rf /opt/jboss/wildfly/standalone/deployments/drools-wb.war.*
+        sudo -u $user touch /opt/jboss/wildfly/standalone/deployments/drools-wb.war.dodeploy
+        ps -eo pid,cmd a | grep 'standalon[e]' | cut -d' ' -f2 | xargs kill -9
```

Comment:
Updated, needed `-f` and since pgrep is clever enough to filter itself out we can drop the `[e]` as well :)